src/index.js

```diff
@@ -7,7 +7,7 @@ const FixedSizeChunker = require('./chunker-fixed-size')
 const through2 = require('through2')
 const UnixFS = require('ipfs-unixfs')
 const async = require('async')
-
+var isNode = !global.window
```

Comment:
do we need some sort of error handling to return a message in case someone tries to use the file or dir importer in a browser context? not sure what is possible and what isn't